### DISCUSSION TITLE
Import ICP-MS chromatograms into mzmine 3

### DISCUSSION THREAD

**Discussion by MarthaGledhill** [mzmine/mzmine#1234] [General]
I am trying to import icp-ms chromatograms into mzmine using the import  "icp-ms cvs" option. Am I right to assume cvs is really csv? I tried it with our chromatography data from a Thermo Element XR, but unfortunately I get a "could not open data file" error - even when I remove all the extra headings and resave as a very plain .csv with just time and intensities or time and intensity from one element with no column heading at all. I am guessing it is something to do with my file format but I ran out of ideas as to how to change the file to get the import to work. What format is required for this option?
Many thanks for any help.

**Comment by robinschmid** [comment]
@SteffenHeu you might be the best to answer this

**Comment by SteffenHeu** [comment]
Hi,

i wrote the import for an Agilent 7800 TQ ICP-MS. It suports data from csv files with this format:

```
D:\Steffen\20200205 - organic tune.b\5351.d
Intensity Vs Time,Counts
Acquired      : 06/02/2020 02:14:15 using Batch 20200205 - organic tune.b
Time [Sec],BaF157 -> 157
0.9999,1510.08
1.9907,1416.07
2.9809,1304.06
3.9716,1421.07
```

**Reply by MarthaGledhill** [reply]
Thanks for following up on this Steffen. When I try to import this after saving it to a csv file I get "Error of task null: Index out of bounds for length 1". I also get the same error when I just copy and paste the first four lines to my own time-intensity data.

**Reply by SteffenHeu** [reply]
Thanks for the report, it is now fixed in the latest development release. you can try it once the macos signing finished.
https://github.com/mzmine/mzmine3/releases it should be newer than "2 h ago"
